### DISCUSSION TITLE
Is there a way to truncate long code  or prevent horizontal scroll on code embeds?

### DISCUSSION THREAD

**Discussion by erbill** [vercel/community#1985] [Help]
### Summary

I'd love to disable horizontal scroll and instead make pasted code readable through vertical scroll.

### Example
```copy
HyperlongtextresultsinhorizontalscrollandhowtopreventitwithoutusingharbrakesHyperlongtextresultsinhorizontalscrollandhowtopreventitwithoutusingharbrakesHyperlongtextresultsinhorizontalscrollandhowtopreventitwithoutusingharbrakes
```
_No response_

**Comment by Abhisha728Kumar** [comment]
Yes In VS code you can use a shortcut key alt+Z key to wrap the text into the visible area

**Reply by erbill** [reply]
but it doesn't reflect in the exported site. is there any way to do this on deployed site?

**Reply by Abhisha728Kumar** [reply]
Hey!, I got your problem that you want to prevent horizontal scrolling for this you are to simply define the width allotted to the text either you allot the specific width or add w-full to the code. I hope this may solve your problem.

**Reply by erbill** [reply]
Thank you for your reply.

w-full would be great I think because I'd love to make it work on mobile as well. 

Where should I add w-full to it? 

currently I just use this formatting;

```js
THIS IS A LONG TEXT PART WHERE I"D LIKE TO HAVE IT BEHAVE LIKE A PHARAGPRAH 
```

I'm trying to make easy to copy text parts, that's why I'm using code snippets. Information boxes like ::: info ::: warps text great but doesn't have a copy text feature. Code snippets has a copy feature but doesn't warp text like default paragraph. 

Any suggestions to make this happen?

thanks!
